### ISSUE TITLE
BSyncr Analysis results should render image results when available

### ISSUE THREAD

**Issue by adrian-lara** [SEED-platform/seed#2535]
**Is your feature request related to a problem? Please describe.**

No

**Describe the solution you'd like**

A BSyncr Analysis can have an associated image as an output file. This output file should be rendered on the analysis results page. 

Design is somewhat flexible, this should render nicely in addition to the BSyncr output file information currently presented:
<img width="1278" alt="Screen Shot 2021-01-08 at 10 26 28 AM" src="https://user-images.githubusercontent.com/30608004/104045619-12f92d00-519c-11eb-8ede-43423183c1bb.png">

**Describe alternatives you've considered**

**Additional context**

Before starting, an example of a relevant BSyncr output/results image would be nice to have, but at minimum, knowing the possible file format(s) of such a file is needed.

**Comment by adrian-lara** [comment]
More consideration may be needed on how other services' results are rendered?

**Comment by macintoshpie** [comment]
This has been implemented:
<img width="1194" alt="Screen Shot 2021-09-29 at 10 57 30 AM" src="https://user-images.githubusercontent.com/18518728/135305414-65e222e6-8468-4ec5-82ef-293b1ef5ed99.png">

SHA: 2.11.0.f3e9a4a4d
